tests/unit/h/tasks/annotations_test.py

```diff
@@ -19,7 +19,7 @@ def test_it(self, factories, annotation_write_service):
         fill_annotation_slim(batch_size=10)
 
         annotation_write_service.upsert_annotation_slim.assert_has_calls(
-            [call(anno) for anno in annos]
+            [call(anno) for anno in annos], any_order=True
```

Comment:
This passed locally and when in CI when I merged the change that removed the guaranteed order. It was only by chance.